azure-pipelines.yml

```diff
@@ -55,40 +57,63 @@ jobs:
         go get -u golang.org/x/lint/golint
       workingDirectory: '$(sdkPath)'
       displayName: 'Install Dependencies'
+
     - script: go vet -v $(go list ./... $(go.list.filter))
       workingDirectory: '$(sdkPath)'
       displayName: 'Vet'
+      env:
+        GO111MODULE: 'off'
```

Comment:
I believe you can add this to the `variables` section so you don't have to repeat it.